migrations/20200629154732-copy-entities-to-elasticsearch.js

```diff
@@ -118,6 +118,41 @@ module.exports = {
 
         entityObj["telemetry_entities"] = telemetryEntities;
 
+        let userData = await db.collection('userExtension').aggregate([
+          {
+              $unwind : "$roles"
```

Comment:
@deepakr123 - Rather than running this old migration, we should write a new migration that will loop through all users in chunks of 100 and update required entities in MongoDB and ES, just in case we have to restore data in ES, MongoDB already has it.

Reply:
resolved